Tools/jit/_writer.py

```diff
@@ -32,8 +32,11 @@ def _dump_footer(
     yield "};"
     yield ""
     yield f"static const void * const symbols_map[{max(len(symbols), 1)}] = {{"
```

Comment:
Looking at this, there shouldn't be a need for the length in here? And while I _believe_ it's okay, it could be affected by this change.